src/include/error.h

```diff
@@ -125,11 +125,15 @@
 
 /*
  * Branch prediction hints. If an expression is likely to return true/false we can use this
- * information to improve performance at runtime.
+ * information to improve performance at runtime. This is only supported for GNU compilers.
```

Comment:
I think we should enable this for Clang/LLVM as well. Or just filter out MSVC since that's the problem child.

Reply:
Good point!
I've pushed the change to only exclude MSVC and ran all the compile jobs in a patch. Everything's succeeding, and if we ever add a new unsupported platform that'll be IDd pretty quickly.